resources/js/Shared/Layout/Nav.vue

```diff
@@ -0,0 +1,61 @@
+<script setup>
+import { ref } from 'vue'
+import { Dialog, DialogPanel } from '@headlessui/vue'
+import { Bars3Icon, XMarkIcon } from '@heroicons/vue/24/outline'
+import { Link } from '@inertiajs/vue3'
+
+const navigation = [
+  { name: 'Cities', href: '/admin/dashboard/cities' },
+  { name: 'Countries', href: '/admin/dashboard/cities' },
+]
+
+const mobileMenuOpen = ref(false)
+</script>
+  
+<template>
+  <header class="w-full bg-white ">
+    <nav class="mx-auto flex h-24 max-w-full items-center justify-between border-b-2 p-6 lg:px-8" aria-label="Global">
+      <Link href="/" class="-m-1.5 flex items-center p-1.5">
+        <img class="h-12 w-auto" src="@/assets/scooter.png" alt="">
```

Comment:
In `src` attribute should path starts with `@`? And `alt` attribute should not be empty.